doc/format-run.md

```diff
@@ -175,6 +175,22 @@ ordinary index type.
 The type of index in use is metadata that must be known for the whole LSM table
 handle. It does not vary per run.
 
+Independently of the concrete index serialization format, the type and the
+version of the format are stored at the beginning as a 32-bit number called the
+type–version indicator. A type–version indicator has the following shape, where
+bit numbers refer to significance:
```

Comment:
This does not tell the reader which number indicates highest significance

Reply:
Well, bits with higher numbers have higher significance. This is the established convention. 🙂 I don’t think that it is necessary to explicitly mention this here.